3.-Web-api-call-Microsoft-graph-for-personal-accounts/README.md

```diff
@@ -174,8 +144,16 @@ What **differs from the previous chapter** is that you will use the same applica
 
 1. On the app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the Visual Studio configuration file for this project (`ida:ClientId` in `TodoListClient\App.Config`).
 1. From the app's Overview page, select the **Authentication** section.
-   - In the **Redirect URIs** list, under **Suggested Redirect URIs for public clients (mobile, desktop)** check the box next to **https://login.microsoftonline.com/common/oauth2/nativeclient**.
-   - Select **Save**.
+   - Click **Add a platform** button.
+   - Select **Mobile and desktop applications** on the right blade.
+   - In the **Redirect URIs** list, check the box next to **https://login.microsoftonline.com/common/oauth2/nativeclient**.
+   - Click **Configure**.
+1. Select the **API permissions** section
+   - Click the **Add a permission** button and then,
+   - Ensure that the **My APIs** tab is selected
```

Comment:
@TiagoBrenck 
Given that this is the same client ID, why would we need at all to expose an API that the same app would call?

Reply:
How would we get `access_as_user` without creating it?